src/PathFinderUtils.js

```diff
@@ -0,0 +1,172 @@
+import Dongle from './Dongle'
+import Networks from './Networks'
+import bitcoin from 'bitcoinjs-lib'
+import bs58 from 'bs58'
+import _ from 'lodash'
+
+console.log(bitcoin)
```

Comment:
a good practice will be to remove the console.log for things that goes to master.